rules/windows/builtin/win_susp_logon_explicit_credentials.yml

```diff
@@ -0,0 +1,31 @@
+title: Suspicous Remote Logon with Explicit Credentials
+id: 941e5c45-cda7-4864-8cea-bbb7458d194a
+status: experimental
+description: Detects suspicious processes logging on with explicit credentials
+references:
+    - https://drive.google.com/file/d/1lKya3_mLnR3UQuCoiYruO3qgu052_iS_/view
+author: 'oscd.community, Teymur Kheirkhabarov @HeirhabarovT, Zach Stanford @svch0st'
+date: 2020/10/05
+tags:
```

Comment:
Hey @svch0stz, nice contribution, but the tags are not fulfilled on your rule, this is causing most of the new PRs tests to fail

Reply:
Fixed that. Thank you, @w0rk3r! Well done spotting that! I hope that now the test will work.
It was our fault that the rule was added to the `oscd` branch. 
We should have spotted that earlier, during the initial review. 
But it seems that tests were already broken at that time, I think it is somehow connected with a number of PRs. 
We experienced the same last year, during the first sprint. 

I hope now it would be OK.